### ISSUE TITLE
After PC crash no connection to Wallet

### ISSUE THREAD

**Issue by Tom29a** [Chia-Network/chia-blockchain#4040]
I use this PC for plotting/farming and mining with one Graphic card. 1x NVME 512GB , 1x SSD 2TB, 2x HDD 14TB
I use latest version 1.1.4

My sun tried to install Minecarft/Java before but due to low space on C it doesnt´t work so we tried on free HDD. 
But Java failed to install on C due to not enough space. So Nicehash mining could crash the pc. After hard close of chia with task manager and restart the pc, i cant connect to wallet again. I let chia try whole night but no success. 

Do you have any tip ? what can i do ?

**Comment by Tom29a** [comment]
![grafik](https://user-images.githubusercontent.com/83813099/117430476-787de000-af28-11eb-8e19-8cbad2a38301.png)

**Comment by xketcuup** [comment]
Are you using GUI (interface) or CLI (comand line) ?
Are you on Windows ? 
I started by using GUI but i'm trying to use CLI. And to me, it's easier on CLI.
You could try to do : 
1. open cmd
2. use "cd C:/Users/[your-user]/AppData/Local/chia-blockchain/app-[your-version]/resources/app.asar.unpacked/daemon"
3. use "chia init" 
4. use "chiat start farmer"
Normally, this would be good.
Optionally, you can do "chia plots check -g k32" (if you created k32 plot)

**Comment by Tom29a** [comment]
result of check:

2021-05-07T11:54:56.633  chia.plotting.check_plots        : WARNING  There are 2 plots with a farmer or pool public key that is not on this machine. The farmer private key must be in the keychain in order to farm them, use 'chia keys' to transfer keys. The pool public keys must be in the config.yaml

i think these two are the ones i plotted with first wallet which i deleted

**Comment by xketcuup** [comment]
> result of check:
> 
> 2021-05-07T11:54:56.633 chia.plotting.check_plots : WARNING There are 2 plots with a farmer or pool public key that is not on this machine. The farmer private key must be in the keychain in order to farm them, use 'chia keys' to transfer keys. The pool public keys must be in the config.yaml
> 
> i think these two are the ones i plotted with first wallet which i deleted ----> should i delete them ?

I'm not an expert but i think you can delete them, even if its not a problem, because as you can see its a "warning" and not an error.
Normally, this comand "chia plots check -g k32", you have to get your number ot plot.

But the real problem is upper with "_[4284] Failed to execute script start_wallet_"

**Comment by Tom29a** [comment]
so should i try to delete my user name chia folder and reinstall chia as discussed in other threat ? 
https://github.com/Chia-Network/chia-blockchain/issues/863#issuecomment-833596038

**Comment by Tom29a** [comment]
so i did this solutions above deleted user/chia folder and restart. Now i have started chia again with public key hopefully i get my plots back again. now synching but i cant see my farm plots by now. plots are running see what happens after synch

**Comment by holdenprkr** [comment]
I have this same traceback error and whenever I run 'chia wallet show' I receive this error

> Connection error. Check if wallet is running at 9256

I also switched to the CLI after the GUI gets stuck loading ever since closing it after the initial installation. The initial installation is the only time that chia has appeared to work properly for me. When I run chia check plots I see the first k-33 plot that finished plotting. I just deleted the k-34 I started plotting on the GUI that never showed as a valid plot in the CLI but nothing has changed. Any help on this would be great, I want to get to farming! :)
